src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/CsvFormatResponseIT.java

```diff
@@ -142,6 +156,460 @@ public void fieldOrderWithScriptFields() throws IOException {
         verifyFieldOrder(expectedFields, query);
     }
 
+    //region Tests migrated from CSVResultsExtractorTests
+
+    @Test
+    public void simpleSearchResultNotNestedNotFlatNoAggs() throws Exception {
+        String query = String.format(Locale.ROOT, "select dog_name,age from %s/dog order by age", TEST_INDEX_DOG);
```

Comment:
A helper method `format` would be helpful so that we don't have to pass `Locale.ROOT` everywhere and also in case of future change on this.

Reply:
that will include pretty large number of files. Do you mind if I do it in a separate PR ?